src/vite.rs

```diff
@@ -164,7 +168,7 @@ impl Production {
             }
         };
         Ok(Self {
-            main: format!("/{}", entry.file),
+            main: entry.clone(),
```

Comment:
We should be able to avoid the `.clone()` call by using [`HashMap::remove`](https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.remove) above.